vip-parsely/vip-parsely.php

```diff
@@ -76,3 +76,22 @@ function (): void {
 		}
 	);
 }
+
+/**
+ * Allows VIP Support users to use the Parse.ly Content Helper feature.
+ *
+ * @param bool    $current_user_can_use_pch_feature Whether the current user can use the Parse.ly Content Helper feature.
+ * @param string  $feature_name The name of the feature.
+ * @param WP_User $current_user The current user.
+ *
+ * @return bool Whether the current user can use the Parse.ly Content Helper feature.
+ */
+add_filter( 'wp_parsely_current_user_can_use_pch_feature', function ( $current_user_can_use_pch_feature, $feature_name, $current_user ) {
+	$user_id = $current_user->ID;
+
+	if ( user_can( $user_id, 'vip_support' ) ) {
+		return true;
+	}
+
+	return $current_user_can_use_pch_feature;
+}, 10, 3 );
```

Comment:
Should we make sure this runs later to ensure that it doesn't get overwritten?

```suggestion
}, 999, 3 );
```

Reply:
Ahh good call!